### PR TITLE
[windows] switch from patching ucrt to directing to fixed version

### PR DESCRIPTION
**Pull request by shuffle2**
DO NOT MERGE
testing linter :)

### CODE REVIEW

**Comment by shuffle2** [comment]
fwiw, the windows buildbot is failing because the buildbot has `api-ms-win-core-version-l1-1-0.dll` but not `api-ms-win-core-version-l1-1-1.dll`, which is the version that the build is currently expecting. (referencing `mincore.lib` just links against whatever is the latest).

I do wonder what is the recommended way to handle this. The optimal solution would be if the system automatically found and installed the updated dependencies, somehow...

**Comment by shuffle2** [comment]
I've changed to linking against `version.lib` (aka the old way). This seems to workaround the issue on the buildbot. However, I think it is actually a problem with the buildbot itself - probably it needs to be rebooted so that updates can finish installing. cc @delroth

File: Source/Core/Common/ucrtFreadWorkaround.cpp [hunk 1]
```diff
@@ -4,93 +4,122 @@
...
+  const wchar_t *const module_names[] = { L"ucrtbase.dll", L"ucrtbased.dll" };
+  for (const auto &name : module_names) {
+    Version version;
+    if (GetModuleVersion(name, &version)) {
+      if (version.build < fixed_build) {
+        auto msg = StringFromFormat("You are running %S version %d.%d.%d.%d.\n"
+          "An important fix affecting dolphin was introduced in build %d.\n"
+          "You can use dolphin, but there will be known bugs.\n"
+          "Please update this file by installing the latest UCRT:\n"
+          "<put_url_here>\n",
```
Comment (JosJuice) [inline Source/Core/Common/ucrtFreadWorkaround.cpp]: This message needs to be translatable. (Using `GetStringT` is probably the best option if you aren't going to be using panic alerts, though I suppose you'll have to replace `MessageBoxA` with `MessageBoxW` and re-encode the string...)

Also, capital D in Dolphin, please.

Comment (shuffle2) [inline Source/Core/Common/ucrtFreadWorkaround.cpp]: Yea, I'll wait to see if this is actually solvable with a download. This PR (at the moment) is mainly for me to test linter.

**Comment by BhaaLseN** [comment]
How the hell did this pass lint with all the same-line curlies? o_o

**Comment by shuffle2** [comment]
@BhaaLseN lint is broken

**Comment by shuffle2** [comment]
@dolphin-emu-bot rebuild

**Pull request closed without merging**